Cognigy/WebchatWidget
# Fix sr-only class scope

This PR changes the 'sr-only' css-rule to a scoped variant, so it is only valid for the webchat widget and not to other parts of the page. 

Success criteria: 
- Sr-only texts are visually hidden as before
- Sr-only texts should still be announced by the screen-readers

How to test:
- Use chromeVox screen reader for testing
- Once the webchat is loaded, wait for the preview message to show up before opening the webchat.
- Move focus to webchat button, if it is not already focused. 
- You should hear 'New message preview', which is an sr-only text, followed by the preview message.
- When you open the webchat and move the focus to the chat history panel (You might want to navigate using Shift + Tab), you should hear the sreen reader announcing 'Chat history'. But, you should not see the heading within the panel, as it is visually hidden by the sr-only class.
